src/nextjournal/clerk/eval.clj

```diff
@@ -163,12 +164,10 @@
     viewers
     (update :nextjournal/viewers eval)))
 
-(defn read+eval-cached [{:as _doc :keys [blob->result ->analysis-info ->hash]} codeblock]
-  (let [{:keys [form vars var]} codeblock
-        {:as form-info :keys [ns-effect? no-cache? freezable?]} (->analysis-info (if (seq vars) (first vars) (analyzer/->key codeblock)))
```

Comment:
There was an ambiguity around under which key a form is stored inside the map `->analysis-info`. This used to cause e.g. #454. The caller of this function now replaces `codeblock` with `(get ->analysis-info (:id codeblock))`